src/debug/HyprCtl.cpp

```diff
@@ -45,6 +45,9 @@ std::string monitorsRequest(HyprCtl::eHyprCtlOutputFormat format) {
         "id": %i,
         "name": "%s"
     },
+    "specialWorkspace": {
```

Comment:
why isnt this just a field

Reply:
The field should have a `name` subfield in the future. I was not sure how to handle the special cases like `specialWorkspaceID == 0` so I didn't add it by myself.

I added a commit in this PR to include the name. PTAL. The output will be like:
```
    "specialWorkspace": {
        "id": -99,
        "name": "special"
    },
```
for a special workspace that's shown or a hidden special workspace like:
```
    "specialWorkspace": {
        "id": 0,
        "name": ""
    },
```